wiki/Guides/Adding_Custom_Hitsounds/en.md

```diff
@@ -2,41 +2,56 @@
 
 From the thread: *[how to add custom hitsound? by: neonat](https://osu.ppy.sh/community/forums/posts/3215699)* (and modified)
 
-## Getting the custom hitsounds
+## Custom Hitsound Files
 
-You'll first need some custom hitsounds! You can either:
-- You can get them from the [Custom Hitsound Library](../Custom_Hitsound_Library/),
-- or make your own
-  - if you do make your own, try to keep the format as `.wav` - as it is best for looping.
+In order to use custom hitsound files in osu!, you'll first need some custom hitsounds! The [Custom Hitsound Library](../Custom_Hitsound_Library/) is a great resource for existing sound samples, including cymbals, drums, bells, whistles and more! Alternatively, if you can't find what you're looking for, you can make your own samples!
 
-Once you get them, move them into the song folder itself. If you don't know where your song folder is located, follow these directions:
+Hitsounds should typically be in the `.wav` or `.ogg` format, as these have no playback delay and loop correctly. `.mp3` formatted sound files have slight delays and may not accurately match up to the sounds in the music you are trying to hitsound. However, they but may still be useable for certain special hitsounds such as applause or ambient noises, where the size of `.wav` or `.ogg` files may be prohibitively large.
```

Comment:
wording is a bit messed up: `However, they but may still...`
should delete the but

Reply:
adding a comma to: `certain, special hitsounds` would be more grammatically correct